README.md

```diff
@@ -51,6 +51,17 @@ Type: `String`
 
 The path to the translation file to be upload to your OneSky project.
 
+#### options.files
+Type: `Array of Object`
+
+```
+[{
+    ProjectId : String, locale : 'String', file : 'String'
+}, {...}]
+```
+
+This list enable you to upload multiple file at a time. Note that you don't need to enter a value for the ProjectId, the locale or the file if you use the Array files. If you do use both at the same time the value you enter for the ProjectId, the locale and the file will add as an additionnal object of the Array files.
```

Comment:
I would not support putting a value for both `file` and `files`. Throw an error if the tasks has both values defined.